src/Symfony/Component/Validator/Constraints/Ip.php

```diff
@@ -64,6 +75,14 @@ class Ip extends Constraint
         self::V4_ONLY_PUBLIC,
         self::V6_ONLY_PUBLIC,
         self::ALL_ONLY_PUBLIC,
+
+        self::V4_ONLY_PRIV,
+        self::V6_ONLY_PRIV,
+        self::ALL_ONLY_PRIV,
+
+        self::V4_ONLY_RES,
+        self::V6_ONLY_RES,
+        self::ALL_ONLY_RES,
```

Comment:
`RES` is super cryptic, let's go with the full `RESERVED` word

Let's duplicate all consts to add these full-word names as aliases (PRIV also)